orca-core/src/main/groovy/com/netflix/spinnaker/orca/pipeline/util/HttpClientUtils.java

```diff
@@ -77,6 +78,13 @@ public long getRetryInterval() {
       }
     );
 
+    httpClientBuilder.setRetryHandler((exception, executionCount, context) -> {
+      HttpUriRequest currentReq = (HttpUriRequest) context.getAttribute(HttpCoreContext.HTTP_REQUEST);
+      Uninterruptibles.sleepUninterruptibly(RETRY_INTERVAL, TimeUnit.MILLISECONDS);
```

Comment:
Yup, the exception here is of type IOException (not obvious since it's in a lambda). 
The API on that method is `boolean retryRequest(IOException exception, int executionCount, HttpContext context);`

We could potentially exclude certain types of IOException i.e `SSLHandshakeException`